ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -222,12 +254,44 @@ public void badInstantiationTest() throws Exception {
     }
   }
 
+  /**
+   * Test HelixClusterManager initialize with null ZNRecord. In such case, HelixClusterManager will initialize replica
+   * state based on instanceConfigs.
+   * @throws Exception
+   */
+  @Test
+  public void emptyPartitionOverrideTest() throws Exception {
+    if (overrideEnabled) {
```

Comment:
can use assumeTrue here and in a couple other places as well

Reply:
done